http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/BaseCloudServiceRuntimeMonitor.scala

```diff
@@ -468,9 +468,11 @@ abstract class BaseCloudServiceRuntimeMonitor[F[_]] {
           )
       }
       // wait for 10 minutes for tools to start up before time out.
-      availableTools <- streamFUntilDone(checkTools,
-                                         monitorConfig.checkToolsMaxAttempts,
-                                         monitorConfig.checkToolsInterval).compile.lastOrError
+      availableTools <- streamFUntilDone(
+        checkTools,
+        monitorConfig.checkTools.maxAttempts,
+        monitorConfig.checkTools.interval
+      ).interruptAfter(monitorConfig.checkTools.interruptAfter).compile.lastOrError
```

Comment:
This is where it interrupts tool polling (fs2 `interuptAfter` also requires `ConcurrentEffect` typeclass).